app/controllers/content_files_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Controller for a Work content file
+class ContentFilesController < ApplicationController
+  # Skipping authorization for now
+  skip_verify_authorized only: %i[show edit update destroy]
+  before_action :set_content_file
+
+  def show; end
+
+  def edit; end
+
+  def update
+    if @content_file.update(content_file_params)
+      redirect_to content_file_path(@content_file)
+    else
+      render :edit, status: :unprocessable_entity
```

Comment:
Shows as uncovered. Worth adding a spec to test the unhappy path?

Reply:
At this point, I don't know how I would I would test, as Content File doesn't have any validation could fail at this point. I'm putting it in proactively should we add validation in the future.

Thoughts?

Reply:
I figure you could stub `update` to return `false` and validate that the form is reloaded? If that's more futzing than value, we can skip it.